libcaf_core/caf/sec.hpp

```diff
@@ -114,6 +114,8 @@ enum class sec : uint8_t {
   bad_function_call = 40,
   /// Feature is disabled in the actor system config.
   feature_disabled,
+  /// Failed to open file to read actor system config.
```

Comment:
That's overly specific. I think the error code should apply to any file, not just the config.